examples/analysis/ubet/README.md

```diff
@@ -0,0 +1,31 @@
+# UBet
+
+In general this directory contains the analysis scripts, configuration, and *most* other things necessary to reproduce our results from the LangSec '23 paper [Automatically Detecting Variability Bugs Through Hybrid Control and Data Flow Analysis](https://langsec.org/spw23/papers.html#variability).
+
+### NITF
+The examples we reference in the paper primarily relate to the [NITF](https://jitc.fhu.disa.mil/projects/nitf/testdata.aspx) (National Imagery Transmission Format) reference parser Nitro, though in our motivation section we also use some specifically targeted toy examples, available under `polytracker/examples/analysis/ubet/examples/motivation` and named by listing.
+
+NITF is a binary image file format. Each NITF packages one or more visual data representations (video, fingerprints, CAT scan, JPEG, etc.) with extra metadata and other conditionally included information e.g., captions, information for rendering visual redactions, or geo-reference data. Nitro parses multiple mutually incompatible versions of the NITF specification. To simulate the effects of encountering a particular bad input we would like reproduce the effects of in a testing, local, or staging environment we applied Nitro instrumented with UBet to a corpus of 148 valid and known-invalid NITF files.
+
+Unfortunately, as also noted in the tags/ subdirectory README, the input files we used to find bugs in Nitro are part of a corpus we accessed during SafeDocs-related evaluation for PolyTracker. As such, we don't have rights to include them here.
```

Comment:
I suggest we leave this section out entirely.

Reply:
this section meaning the NITF section? or the NITF standard section? both?